teamA/leeChanWoo/app.js

```diff
@@ -0,0 +1,36 @@
+const http = require('http');
+const express = require('express');
+const morgan = require('morgan');
+const cors = require('cors');
+const dotenv = require('dotenv').config();
```

Comment:
해당 부분 아래와 같이 수정해 주시고, 최상단으로 올려주세요 !
```suggestion
require('dotenv').config();
```